joern-cli/frontends/c2cpg/src/test/scala/io/joern/c2cpg/passes/ast/MethodTests.scala

```diff
@@ -330,6 +330,55 @@ class MethodTests extends C2CpgSuite {
       method.fullName shouldBe "method"
     }
 
+    "be correct for C++ method with long int types" in {
+      val cpg = code(
+        """
+          |long methodA(long int a, long int b) { return a *  b; }
+          |long int methodB(long a, long b) { return a *  b; }
+          |int main() {
+          |  methodA(1.9, 2.9);
+          |  methodB(1.9, 2.9);
+          |  return 0;
+          |}
+          |""".stripMargin,
+        "test.cpp"
+      )
+      val List(methodA) = cpg.method.nameExact("methodA").l
+      methodA.signature shouldBe "long(long,long)"
+      methodA.fullName shouldBe "methodA:long(long,long)"
+      val List(methodB) = cpg.method.nameExact("methodB").l
+      methodB.signature shouldBe "long(long,long)"
+      methodB.fullName shouldBe "methodB:long(long,long)"
+    }
+
+    "be correct for C++ method in class" in {
+      val cpg = code(
+        """
+          |#include <vector>
+          |
+          |class A {
+          |  public:
+          |    int foo1(void) { return; }
```

Comment:
```suggestion
          |    int foo1(void) { return 1; }
```

I was confused for a bit there thinking this was a `void` function